frontend/src/components/utils/Footer/index.tsx

```diff
@@ -16,24 +18,42 @@ const useStyles = makeStyles(() => ({
       paddingRight: '1em',
     },
   },
-  link: {
+  contact: {
     color: '#B94630',
     fontWeight: 'bold',
     textAlign: 'center',
+    marginLeft: '4px',
   },
 }));
 const Footer = (): ReactElement => {
-  const { footer, link } = useStyles();
+  const { footer, contact } = useStyles();
   return (
     <footer>
-      <Grid container alignItems="center" justify="center" className={footer}>
+      <Grid container alignItems="center" justify="center" className={footer} direction="column">
         <Grid item>
-          <Typography>
-            Got a question that isn’t answered?{' '}
-            <Link className={link} underline="none" href="mailto:hello@cornelldti.org">
-              Contact us!
-            </Link>
-          </Typography>
+          <Grid container alignItems="center" justify="center">
+            <Grid item>
+              <Typography>Got a question that isn’t answered? </Typography>
+            </Grid>
+            <Grid item>
+              <Typography>
+                <Link className={contact} href="mailto:hello@cornelldti.org">
+                  Contact us!
+                </Link>
+              </Typography>
+            </Grid>
+          </Grid>
+        </Grid>
+        <Grid item>
+          <Link
+            {...{
+              to: `/policies`,
+              style: { color: 'black' },
+              component: RouterLink,
+            }}
+          >
+            <Typography> Privacy Policy </Typography>
```

Comment:
This seems to not line up with the rest of the footer for certain screen sizes on mobile, particularly for the iPhone 5/SE and the Galaxy Fold. See the screenshots below:
<img width="380" alt="Screen Shot 2021-11-07 at 11 45 34 PM" src="https://user-images.githubusercontent.com/26571581/140685940-96f4643b-fc45-4164-8821-e01de33c3ab3.png">
<img width="380" alt="Screen Shot 2021-11-07 at 11 45 42 PM" src="https://user-images.githubusercontent.com/26571581/140685949-eff7b60d-a76d-4b13-9eaa-66ce1fe7a90e.png">
Can we fix this so that it lines up for the smaller screen sizes?

Reply:
I can't figure out how to fix it on super small screen sizes e.g. Galaxy Fold. But I got it to work on all other sizes. 
<img width="527" alt="Screen Shot 2021-11-08 at 10 26 43 AM" src="https://user-images.githubusercontent.com/57203639/140769878-d3f6fff5-5cf2-4e31-81b7-491db5a8e54a.png">

Reply:
Yeah that sounds good to me!